build/Settings.cs

```diff
@@ -23,7 +23,7 @@ private static string config(string @default = null, [CallerMemberName] string k
         public const string DotnetIsolatedProjectTemplatesVersion = "4.0.2288";
         public const string DotnetItemTemplatesVersion = "4.0.2185";
         public const string DotnetProjectTemplatesVersion = "4.0.2185";
-        public const string TemplateJsonVersion = "3.1.1648";
+        public const string TemplateJsonVersion = "3.1.1648"; // todo: Update here with the ext bundle v3 template version. The one which has v2 templates including templates-v2 and bindings-v2 folder. 
```

Comment:
We have to update with correct version which has v2 templates after ext bundle is released with v2 templates. Please see the todo message for details.